.gitignore

```diff
@@ -10,4 +10,5 @@ project/boot
 project/build/target
 project/plugins/lib_managed
 project/plugins/src_managed
-project/plugins/target
\ No newline at end of file
+project/plugins/target
+.idea
```

Comment:
This should rather go into your dev machine's global .gitignore, not this project-level file.
(The comment at the top of this file has more details...)

Reply:
Removed.